Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py

```diff
@@ -131,16 +123,14 @@ def add_particles():
 # simulation run
 ##########################
 
-sim.step(max_steps - 1, mpi_comm=new_comm)
```

Comment:
Oh, where did the comm go? :)

Do we still cover external communicators in other tests? :)

Reply:
Yeap that is covered in test `Python_pass_mpi_comm`.